README.md

```diff
@@ -57,9 +57,9 @@ jobs:
         timeout: 8m
     
     - name: comment PR
-      uses: okteto/notify-pr@latest
+      uses: okteto/notify-pr@v1.0
```

Comment:
@wasabigeek thanks for taking care of this. could you use `master` as the label instead?